src/main/scala/com/gu/mobile/content/notifications/lib/ContentAlertPayloadBuilder.scala

```diff
@@ -0,0 +1,95 @@
+package com.gu.mobile.content.notifications.lib
+
+import java.net.URI
+
+import com.gu.mobile.content.notifications.lib.Seqs._
+import com.gu.mobile.content.notifications.lib.ContentApi._
+import com.gu.contentapi.client.model.v1._
+import com.gu.mobile.content.notifications.Config
+import com.gu.mobile.notifications.client.models.TopicTypes.{ TagBlog, TagContributor, TagKeyword, TagSeries }
+import com.gu.mobile.notifications.client.models._
+
+import scala.util.Try
+
+trait ContentAlertPayloadBuilder {
+
+  val Sender = "mobile-notifications-content"
+  val config: Config
+
+  private val topicsWithoutPrefix = Set(
+    Topic(TagSeries, "membership/series/weekend-reading"),
```

Comment:
I think we now got a couple more here: https://github.com/guardian/mobile-notifications-content/blob/84f13d5d38c5a52eb5c928b7497959f25a22ba3e/src/main/scala/com/gu/mobile/notifications/content/lib/ContentAlertPayloadBuilder.scala#L23-L28